src/GroupHeader.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import CustomPropTypes from './util/propTypes';
+
+function GroupHeader(props) {
+  const {
```

Comment:
@yoyis3000 I usually try to hold off on destructuring things until I have to reference them more than once in the enclosing scope.